app/models/spree/print_invoice_configuration.rb

```diff
@@ -1,8 +1,11 @@
 module Spree
   class PrintInvoiceConfiguration < Preferences::Configuration
-
     preference :print_invoice_next_number, :integer, :default => nil
-    preference :print_invoice_logo_path, :string, :default => Spree::Config[:admin_interface_logo]
+
```

Comment:
Would you mind leaving a comment to explain why we are doing this?

Reply:
Sure thing!

Reply:
Done @jacobherrington! WDYT?